programs/clearing_house/src/controller/orders.rs

```diff
@@ -935,14 +935,13 @@ pub fn fulfill_order_with_amm(
         slot,
     )?;
 
-    let (order_direction, order_post_only) =
-        get_struct_values!(user.orders[order_index], direction, post_only);
-
     if base_asset_amount == 0 {
         msg!("Amm cant fulfill order");
         return Ok((0, false));
     }
 
+    let (_, order_post_only) = get_struct_values!(user.orders[order_index], direction, post_only);
```

Comment:
you can just get rid of the `direction` argument and then you don't need to do `_`